src/bun.js/api/BunObject.zig

```diff
@@ -3436,6 +3438,26 @@ pub fn serve(
     unreachable;
 }
 
+extern fn Bun__setChannelRef(*JSC.JSGlobalObject, bool) void;
+pub fn setRef(globalObject: *JSC.JSGlobalObject, callframe: *JSC.CallFrame) JSC.JSValue {
```

Comment:
This should be in a file related to IPC/node:cluster and it should not be Bun.setRef.